packages/shopping/src/__tests__/acceptance/user.controller.acceptance.ts

```diff
@@ -238,21 +241,33 @@ describe('UserController', () => {
 
   describe('user product recommendation (service) api', () => {
     let recommendationService: HttpServer;
+    let recommendationGRPCService: Server;
 
     before(async () => {
       recommendationService = createRecommendationServer();
       await recommendationService.start();
+      recommendationGRPCService = createGRPCRecommendationServer();
+      recommendationGRPCService.start();
```

Comment:
Do we need `await` for this line as well?

Reply:
No. This start does not return a promise.